index.html

```diff
@@ -1,75 +1,425 @@
+            &#160;Open tools for the analysis and visualisation of Earth science
+            data
+          </h3>
+          <p class="mbr-text align-center pb-3 mbr-fonts-style display-5">
+          SciTools is a collaborative effort to produce powerful Python-based
+          open-source tools for Earth scientists. Initially started at the
+          <a href="https://www.metoffice.gov.uk/"
+             target="_blank"
+             class="text-white">Met Office</a> in 2010, SciTools has grown
+             into a diverse community of partners and collaborators from
+             around the world. SciTools is responsible for the maintenance of
+             a number of key packages such as Iris and Cartopy, and continues
+             to develop new and innovate tools for the Earth scientist&#39;s
+             toolkit.</p>
+          <div class="mbr-section-btn align-center">
+            <a class="btn btn-md btn-primary display-4" href="index.html#iris">
+                <span class="mbri-globe-2 mbr-iconfont mbr-iconfont-btn">
+                </span> Iris
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#cartopy">
+                <span class="mbri-globe mbr-iconfont mbr-iconfont-btn">
+                </span> Cartopy
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#additional_tools">
+                <span class="mbri-setting3 mbr-iconfont mbr-iconfont-btn">
+                </span> Additional Tools
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#cf_units">
+                <span class="mbri-watch mbr-iconfont mbr-iconfont-btn">
+                </span> cf_units
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="features2 iris-cartopy-box">
+    <div class="container">
+      <div class="media-container-row">
+        <div class="card p-3 col-12 col-md-6">
+          <a class="anchor" name="iris" id="iris"></a>
+          <div class="card-wrapper">
+            <div class="card-img"><img src="static/images/iris-1046x697.png"
+                 alt="iris-logo"
+                 title=""></div>
+            <div class="card-box">
+              <h4 class="card-title pb-3 mbr-fonts-style display-7"><a href=
+              "http://scitools.org.uk/iris/docs/latest/index.html"
+                 target="_blank">Iris</a></h4>
+              <p class="mbr-text mbr-fonts-style display-7">The <a href=
+              "http://scitools.org.uk/iris/docs/latest/index.html"
+                 target="_blank">Iris</a> library implements a data model to
+                 create a data abstraction layer which isolates analysis and
+                 visualisation code from data format specifics. The data model
+                 we have chosen is the CF Data Model. The implementation of
+                 this model we have called an Iris Cube.<br>
```

Comment:
These sentences are difficult to read and not as accessible as a summary should be, considering people may not familiar with these concepts
I will try to think of a way to reword this

Reply:
Agree completely. I've wanted to do this too. I think whatever we write, we should also put in the iris README.md/rst too.